File: src/main/java/com/example/dpop/orchestrator/api/v1/ChannelSessionRequest.java
package com.example.dpop.orchestrator.api.v1;

import com.fasterxml.jackson.annotation.JsonInclude;

import java.util.Map;

@JsonInclude(JsonInclude.Include.NON_NULL)
public record ChannelSessionRequest(
        String channel,
        Map<String, Object> data
) {}

File: src/main/kotlin/com/example/dpop/orchestrator/api/v1/ChannelSessionRequest.kt
package com.example.dpop.orchestrator.api.v1

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
@JvmRecord
data class ChannelSessionRequest(
    val channel: String?,
    val data: Map<String, Any>?
)
